src/core/Util.js

```diff
@@ -156,8 +156,8 @@ export function template(str, data) {
 // mobile devices (by setting image `src` to this string).
 export const emptyImageUrl = 'data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=';
 
-const requestFn = window.requestAnimationFrame;
-const cancelFn = window.cancelAnimationFrame;
+const requestFn = typeof window === 'undefined' ? falseFn : window.requestAnimationFrame;
+const cancelFn = typeof window === 'undefined' ? falseFn : window.cancelAnimationFrame;

```

Comment:
@jonkoops I don't think we should be that thorough here — IMO we should do the minimum possible change that makes the bundle itself execute without errors, disregarding how it will be used in such environment. Putting too much guards just bloats the code for cases no one will run into.

Reply:
Fair, just want to prevent that in this case things become polymorphic. In this case `function` and `undefined`.